nearswap/src/lib.rs

```diff
@@ -363,9 +377,17 @@ impl NearSwap {
             tokens_out,
             near_swapped,
         );
+        self.storage_check(start_storage);
         return tokens_in_to_pay.into();
     }
 
+    pub fn storage_check(&mut self, storage: StorageUsage) {
```

Comment:
Let's use this interface
```
    pub fn unsafe_storage_check(&mut self, start_storage: StorageUsage) {
```

Reply:
1. Must be a private function
2. add doc string to explain that this is unsafe when we have other deposit instance around this function call, which can overwrite changes here.

Reply:
ok